### ISSUE TITLE
`ilab train` shouldn't go away

### ISSUE THREAD

**Issue by RobotSail** [instructlab/instructlab#1419]
I tried running `ilab train` and got this error:
```
You are using an aliased command, this will be deprecated in a future release. Please consider using `ilab model train` instead
```

We shouldn't get rid of `ilab train` since it's intuitive to call and should just keep it aliased.

**Comment by nathan-weinberg** [comment]
I don't think this is a really an option - all the aliases commands are meant to be going away and replaced by the model engine commands. Maintaining both is a waste of effort IMHO.

**Comment by leseb** [comment]
We decided as a group all the aliases would be removed in 0.19, which was marked in our CHANGELOG.md.

**Comment by nathan-weinberg** [comment]
@cdoern you want to weigh in here?
